semantic-release/action.yaml

```diff
@@ -96,7 +100,7 @@ runs:
         semantic-release \
           --branches ${{ inputs.release-branch }} \
           --plugins "@semantic-release/commit-analyzer,@semantic-release/release-notes-generator,@semantic-release/git,@semantic-release/github${PLUGINS}" \
-          --tag-format \${version} \
+          --tag-format ${{ inputs.release-prefix }}\${version} \
```

Comment:
Does it make sense to just append the prefix without any separation? I could not imagine a case where we use a prefix without a separator..

Reply:
I assume that the separator is coming from the property `release-prefix`. Otherwise we would have to build a condition so that the separator is not added if `release-prrefix` is not specified.